docs/snapshotters/README.md

```diff
@@ -17,6 +17,7 @@ Block-based:
 Filesystem-specific:
 - `btrfs`: btrfs. Needs the plugin root (`/var/lib/containerd/io.containerd.snapshotter.v1.btrfs`) to be mounted as btrfs.
 - `zfs`: ZFS. Needs the plugin root (`/var/lib/containerd/io.containerd.snapshotter.v1.zfs`) to be mounted as ZFS. See also https://github.com/containerd/zfs .
+- `erofs`: EROFS. `OverlayFS` kernel module needs to be enabled for active snapshots. See also [`erofs.md`](./erofs.md).
```

Comment:
I'm not sure we are going to inflate the number of the built-in snapshotters.
Would it be possible to implement this as a non-core snapshotter plugin?

Reply:
I'm not sure why you're insisted on this, could you elaborate your reasons why EROFS needs to be a non-core snapshotter plugin?
- First, `erofs` is a Linux native kernel filesystem, just like `btrfs` and `devmapper`, and I believe `erofs` should be at least more useful to `devmapper;
- Second, there are enough discussions to make EROFS used more widely.

Reply:
Add more words, I believe it shouldn't have extra maintainence overhead (which, I think, is the most concern stuff for various opensource communities to keep something out-of-tree) and endless unneeded complexity just because of `erofs` snapshotter since it will always use snapshotter and differ APIs in the foreseeable future.
Also, it's unlike `aufs` (which has never landed in kernel) and `zfs` (I think it's just because of its license issue for kernel too.)

Reply:
I hope it's resolved. Anyway, `erofs` will consistently develop useful features for container ecosystems, at least as much as I can. So I resolve this conversation.